plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/FlexDeployPreferencesPanelTest.java

```diff
@@ -0,0 +1,32 @@
+package com.google.cloud.tools.eclipse.appengine.deploy.ui;
```

Comment:
sorry, I've added licenses to 4 other files, but not to these. I've just added to these as well.